pkg/cmd/server/bootstrappolicy/securitycontextconstraints_test.go

```diff
@@ -2,20 +2,22 @@ package bootstrappolicy
 
 import (
 	"reflect"
+	"sort"
 	"testing"
 
 	"k8s.io/apiserver/pkg/authentication/serviceaccount"
 
 	securityapi "github.com/openshift/origin/pkg/security/apis/security"
+	scc "github.com/openshift/origin/pkg/security/securitycontextconstraints"
 )
 
 func TestBootstrappedConstraints(t *testing.T) {
 	expectedConstraints := []string{
 		SecurityContextConstraintPrivileged,
-		SecurityContextConstraintRestricted,
 		SecurityContextConstraintNonRoot,
 		SecurityContextConstraintHostMountAndAnyUID,
 		SecurityContextConstraintHostNS,
+		SecurityContextConstraintRestricted,
```

Comment:
Not according to my running this on master. I specifically made this completely separate commit on top of master, so that we get to agree on the proper semantics first.